app/assets/javascripts/application/open_all.js.coffee

```diff
@@ -0,0 +1,7 @@
+jQuery ->
+
+$("#open_all").click ->
+
+  imports = $(this).data('bgeigie');
```

Comment:
I'd call this "bgeigie_imports". Looks like `imports` might be some kind of reserved word based on how it's highlighting

Reply:
Maybe, is it because of variable declaration? On line 6, `imports` is not highlighted.

Anyway, `import` is indeed reserved word in ES6 (ES2015) or later, so it might be better to avoid using `import` or `imports`.